src/datachain/lib/dc/datasets.py

```diff
@@ -112,7 +110,8 @@ def datasets(
         session: Optional session instance. If not provided, uses default session.
         settings: Optional dictionary of settings to configure the chain.
         in_memory: If True, creates an in-memory session. Defaults to False.
-        object_name: Name of the output object in the chain. Defaults to "dataset".
+        column: Name of the output object in the chain. Defaults to None which
```

Comment:
this description (column is an object) looks weird ...  if we are renaming this, we should also change the description I think